fmriprep/workflows/epi.py

```diff
@@ -138,6 +138,17 @@ def ref_epi_t1_registration(reportlet_suffix, inv_ds_suffix, name='ref_epi_t1_re
 
     explicit_mask_epi = pe.Node(fsl.ApplyMask(), name="explicit_mask_epi")
 
+    if settings['freesurfer']:
+        bbregister = pe.Node(
+            BBRegisterRPT(
+                contrast_type='t2',
+                init='header',
+                registered_file=True,
+                out_fsl_file=True,
```

Comment:
We should definitely make reportlets work independently off what `remove_unnecessary_outputs` is set to.

As far as I remember with DEBUG logging you get information which files are removed.